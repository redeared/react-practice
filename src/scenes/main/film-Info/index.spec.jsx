import TestRenderer from 'react-test-renderer';
import React from 'react';
import { FilmInfo } from './index';

describe('FilmInfo', () => {
    it('should render without crash', () => {
        const comp = TestRenderer.create(<FilmInfo film={{ title: 'some' }} />).toJSON();
        expect(comp).toMatchSnapshot();
    });
});
