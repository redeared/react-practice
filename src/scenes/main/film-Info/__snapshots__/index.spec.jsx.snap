// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilmInfo should render without crash 1`] = `
<div
  style={
    Object {
      "background": "rgba(0, 0, 0, 0.7)",
      "color": "white",
      "paddingLeft": "1em",
    }
  }
>
  <div>
    <button
      onClick={[Function]}
    >
      SEARCH
    </button>
  </div>
  <img
    alt="some"
    className="img"
    src=""
  />
  <div>
    SOME
      
    <span>
      10/10
    </span>
  </div>
  <div>
     
      
     
     min
  </div>
  <div />
</div>
`;
